src/cli/subcommands/archive_cmd.rs

```diff
@@ -48,19 +58,134 @@ pub enum ArchiveCommands {
         /// Path to an uncompressed archive (CAR)
         snapshot: PathBuf,
     },
+    /// Trim a snapshot of the chain and write it to `<output_path>`
+    Export {
+        /// Snapshot input path. Currently supports only `.car` file format.
+        #[arg(index = 1)]
+        input_path: PathBuf,
+        /// Snapshot output filename or directory. Defaults to
+        /// `./forest_snapshot_{chain}_{year}-{month}-{day}_height_{epoch}.car.zst`.
+        #[arg(short, default_value = ".", verbatim_doc_comment)]
+        output_path: PathBuf,
+        /// Latest epoch that has to be exported for this snapshot, the upper bound. This value
+        /// cannot be greater than the latest epoch available in the input snapshot.
+        #[arg(short)]
+        epoch: ChainEpoch,
+        /// How far back we want to go. Think of it as `$epoch - $depth`, the lower bound of this
+        /// snapshot. This value cannot be less than `chain finality`, which is currently assumed
+        /// to be `900`. If this ever changes - the actual value is specified in the error message
+        /// that is thrown in case `depth` value is too low.
+        /// This parameter is optional due to the fact that we need to fetch the exact default
+        /// dynamically from configuration.
+        // TODO: Investigate if we can have a dynamic default here somehow.
```

Comment:
Nit: I have seen others suggest linking to an issue instead of having an orphan `TODO` in the code.

Reply:
Good one, I will do that.